src/main/java/uk/gov/justice/hmpps/prison/service/MovementsService.java

```diff
@@ -181,7 +181,7 @@ public int getEnrouteOffenderCount(final String agencyId, final LocalDate date)
         return movementsRepository.getEnrouteMovementsOffenderCount(agencyId, defaultedDate);
     }
 
-    @VerifyAgencyAccess(overrideRoles = {"SYSTEM_USER"})
```

Comment:
These 2 methods are only called by prison-staff-hub which does not have SYSTEM_USER role